fileinfo.go

```diff
@@ -36,6 +38,8 @@ func SetFileBasicInfo(f *os.File, bi *FileBasicInfo) error {
 	if err := setFileInformationByHandle(syscall.Handle(f.Fd()), fileBasicInfo, (*byte)(unsafe.Pointer(bi)), uint32(unsafe.Sizeof(*bi))); err != nil {
 		return &os.PathError{Op: "SetFileInformationByHandle", Path: f.Name(), Err: err}
 	}
+	runtime.KeepAlive(f)
+	runtime.KeepAlive(bi)
```

Comment:
unsafe.Pointer(x) and (*byte)(x) are casts, not function calls, so they're presumably handled differently. So I don't think that's true.